src/utils/themes-manager.js

```diff
@@ -30,11 +30,13 @@ const dynamicColoredIconsSvgUrlVars = [
 export class ThemesManager {
   player: Object;
   config: Object;
+  playerContainerElement: HTMLElement | null;
```

Comment:
```suggestion
  playerContainerElement!: HTMLElement;
```
after you add this operator you will be able to get rid of the **?** operator below